webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/NetworkingInterfaces/NICDescriptionList.js

```diff
@@ -0,0 +1,132 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import {
+  Label,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Button,
+  Grid,
+  GridItem,
+  Accordion,
+  AccordionItem,
+  AccordionToggle,
+  AccordionContent,
+} from '@patternfly/react-core';
+import { TagIcon, HddIcon } from '@patternfly/react-icons';
+import { translate as __ } from '../../../../../../common/I18n';
+import { STATUS } from '../../../../../../constants';
+
+const types = {
+  interface: __('Interface'),
+  bridge: __('Bridge'),
+  bmc: __('BMC'),
+};
+const NICDescriptionList = ({ status, foremanInterface }) => {
+  const {
+    name,
+    primary,
+    provision,
+    type,
+    mac,
+    mtu,
+    ip,
+    ip6,
+    fqdn,
+    subnet_id: subnetID,
+    subnet_name: subnetName,
+    subnet6_id: subnet6ID,
+    subnet6_name: subnet6Name,
+  } = foremanInterface;
+  const [isExpanded, setIsExpanded] = useState(primary);
+  return (
+    <Accordion className="interface-accordion">
+      <AccordionItem>
+        <AccordionToggle
+          onClick={() => {
+            setIsExpanded(curr => !curr);
+          }}
+          isExpanded={isExpanded}
+        >
+          <Grid>
+            <GridItem span={7} className="interface-name">
+              {name || <span className="disabled-text">{__('No name')}</span>}
```

Comment:
Shouldn't this say, for example, "eth0" instead of the FQDN? That's what it says on the old UI. And it would help with the crowding.

Reply:
yeah, I think this should be the "Device identifier" instead.

Reply:
Nice catch, fixed :)